js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
Hmmm… I thought that we could use the featured editor, but disable all html-like features. Also I'd like to have the checkbox in the editor's tools menu. Would that be possible with trumbowyg?